tools/fuzz/fuzz_targets/fuzz_parse_sql.rs

```diff
@@ -12,13 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use databend_query::sessions::Session;
+use databend_query::sessions::SessionType;
 use databend_query::sql::DfParser;
 use honggfuzz::fuzz;
 
 fn main() {
     loop {
         fuzz!(|data: String| {
-            let _ = DfParser::parse_sql(&data);
+            let _ = DfParser::parse_sql(&data, SessionType::Fuzz);
```

Comment:
Hmm, how to run fuzz?

Reply:
Actually, I also have some issues with this file. I'm not sure why we need this file. And can not find where it is used.

To build success, I add a new SessionType Fuzz.

Reply:
Hi @PsiACE Are we using this for testing now? And how can I use it for databend query?

Reply:
Fuzz type is not necessary, I think it makes more sense to use it for fuzz testing of existing session types. As for how to run the fuzz test, you can see https://github.com/datafuselabs/databend/blob/main/tools/fuzz/README.md

It is not integrated in CI for now, and I am not sure if there are plans to use it to improve our parser. We have introduced the fuzz test in #700. It has helped us in the past to make some small improvements to the parser or the upstream sqlparser.